agent/bench-scripts/tests/pbench-user-benchmark/test-03.pre

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Remove the mocked start/stop tool scripts
+rm ${_testopt}/unittest-scripts/pbench-*-tools 
```

Comment:
For all other `bench-scripts` tests we use mocks for start/stop tools, etc.  But for this one test, `test-03`, we remove the mocks so that we can be sure they execute properly against a tool group with no registered tools.